### PR TITLE
 Allowed Hotkey without using Command

### PR DESCRIPTION
**Pull request by workgroupengineering**
## What does the pull request do?

 Allowed Hotkey without using Command

## What is the current behavior?

Click is not invoke when press Hotkey if with subcribe Click event without set Command.  

## What is the updated/expected behavior with this PR?

Click event is invoke on Hotkey

## Checklist

- [x] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0018379-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0018436-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019457-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019679-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.Controls/HotkeyManager.cs [hunk 1]
```diff
@@ -12,21 +12,61 @@ public class HotKeyManager
...
             }
 
-            public readonly ICommandSource? CommandSource;
+            public ICommand? GetCommand()
+            {
+                if (reference.Target is { } target)
+                {
+                    if (target is ICommandSource commandSource && commandSource.Command is { } command)
+                    {
+                        return command;
```
Comment (jinek) [inline src/Avalonia.Controls/HotkeyManager.cs:28]: Am I right that Click event will stop responding to a hotkey if a command set on a button?

Comment (workgroupengineering) [reply src/Avalonia.Controls/HotkeyManager.cs:28]: > Am I right that Click event will stop responding to a hotkey if a command set on a button?

Only when CanExecute return false

> Is that how it works in WPF?

If I remember correctly it's the same behavior as WPF.

**Pull request merged**